ext/gd/gd.c

```diff
@@ -3822,6 +3848,22 @@ PHP_FUNCTION(imageantialias)
 }
 /* }}} */
 
+static bool _php_gd_zval_try_get_c_int(zval *tmp, const char *field, int *res) {
```

Comment:
Symbols starting with `_` are reserved in C, so please avoid this.
```suggestion
static bool php_gd_zval_try_get_c_int(zval *tmp, const char *field, int *res) {
```

Reply:
sorry just saw your comment now, agreed.